README.md

```diff
@@ -142,68 +119,112 @@ make undeploy-kind
+make docker-push-daemon DAEMON_IMG=<some-registry>/ingress-node-firewall-daemon:latest
+# or make podman-push-daemon DAEMON_IMG=<some-registry>/ingress-node-firewall-daemon:latest
 ```
-6. Push controller container image to an image registry
+6. Set the daemon image name
 ```sh
-make docker-push-daemon DAEMON_IMG=<some-registry>/ingress-node-firewall-daemom:latest
+hack/set-daemon-image.sh <some-registry>/ingress-node-firewall-daemon:latest
 ```
-7. Manually edit "config/manager/env.yaml" and add the daemon image name to value of environment variable "DAEMONSET_IMAGE"
-8. make deploy IMG=<some-registry>/ingress-node-firewall-controller:latest
-9. Undeploy resources from OCP cluster
+
+#### Deploy from manifests
+
+7. Install custom resource definitions
+```sh
+make install
+```
+8. Deploy resources to OpenShift cluster
+```sh
+make deploy IMG=<some-registry>/ingress-node-firewall-controller:latest
+```
+
+##### To uninstall
+
+Undeploy resources from OCP cluster
 ```sh
 make undeploy
 ```
-10. Uninstall custom resource definitions
+Uninstall custom resource definitions
 ```sh
 make uninstall
 ```
-### Deploy with OLM
-2. Build and push bundle and index images to an image registry
+
+#### Deploy with OLM
+
+7. Build and push bundle and index images to an image registry. 
 ```sh
-make build-and-push-bundle-images IMG=<some-registry>/ingress-node-firewall-controller:latest DAEMON_IMG=<some-registry>/ingress-node-firewall-daemom:latest IMAGE_TAG_BASE=<some-registry>/ingress-nodefw/ingress-node-firewall
+make build-and-push-bundle-images \
+  IMG=<some-registry>/ingress-node-firewall-controller:latest \
+  BUNDLE_IMG=<some-registry>/ingress-node-firewall-bundle:latest \
+  BUNDLE_INDEX_IMG=<some-registry>/ingress-node-firewall-index:latest
+# or make podman-build-and-push-bundle-images \
+#      IMG=<some-registry>/ingress-node-firewall-controller:latest \
+#      BUNDLE_IMG=<some-registry>/ingress-node-firewall-bundle:latest \
+#      BUNDLE_INDEX_IMG=<some-registry>/ingress-node-firewall-index:latest
 ```
-3. Deploy with OLM
+
+8. Deploy with OLM
 ```sh
-make deploy-with-olm NAMESPACE=openshift-ingress-node-firewall IMAGE_TAG_BASE=<some-registry>/ingress-nodefw/ingress-node-firewall
+make deploy-with-olm \
+  NAMESPACE=openshift-ingress-node-firewall \
+  BUNDLE_INDEX_IMG=<some-registry>/ingress-node-firewall-index:latest
 ```
-4. Undeploy resources from OCP cluster
+
+##### To uninstall
+
+Undeploy resources from OCP cluster
 ```sh
 oc delete ns openshift-ingress-node-firewall
 ```
-5. Uninstall custom resource definitions
+Uninstall custom resource definitions
 ```sh
 make uninstall
 ```
 
-## Disable webhook
-Remove manager binary flag `--enable-webhook` from the containers command in file config/manager/manager.yaml
+## Testing
+
+### Running test
+
+To run ingress-node-firewall-operator unit tests (no cluster required), execute the following:
+```shell
+make test
```

Comment:
there is also `make test-race` to check for race conditions
in general `make help` should list all commands